letsencrypt/client/client.py

```diff
@@ -339,7 +339,10 @@ def init_key():
     the namedtuple to easily work with the protocol.
 
     """
-    key_pem = crypto_util.make_key(CONFIG.RSA_KEY_SIZE)
+    if key_size < CONFIG.RSA_KEY_SIZE:
+        logging.warning("Generating keys smaller than %d bits is NOT recommended!", CONFIG.RSA_KEY_SIZE)
+
+    key_pem = crypto_util.make_key(key_size)
```

Comment:
OK, so I'll add an exception handler for ValueError and remove the check for key_size completely?